README.md

```diff
@@ -0,0 +1,36 @@
+# openvpn-client
+
+This role installs and copys an openvpn configuration, either from the [openvpn server](https://github.com/stuvusIT/openvpn) or from files in the inventory
+
+
+## Requirements
+
+An openvpn config file either on the openvpn server or in the inventory and an apt based system.
+
+## Role Variables
+
+| Variable                          | Default / Mandatory        | Description                                                                                |
+|-----------------------------------|----------------------------|--------------------------------------------------------------------------------------------|
+| `openvpn_client_server`           | :heavy_multiplication_x:   | Inventory name of the host running the [openvpn role](https://github.com/stuvusIT/openvpn) |
```

Comment:
Please state how the config file on the server has to be named and where this role will search for it.

Reply:
no i clearly state that you have to use my role to do this so this is irrelevant